guides/metrics.md

```diff
@@ -0,0 +1,25 @@
+# How to configure metrics
+
+# Metrics
+
+Rollkit can report and serve the Prometheus metrics, which in their turn can
+be consumed by Prometheus collector(s).
+
+This functionality is disabled by default.
+
+To enable the Prometheus metrics, set `instrumentation.prometheus=true` in your
+config file. Metrics will be served under `/metrics` on 26660 port by default.
```

Comment:
Maybe a dumb question - but which file exactly?

Reply:
rollkit.toml?

Reply:
Actually this is the cometbft app's `config.toml`. Will clarify the location 👍🏽